### PR TITLE
[Application] Update yolo v2 model similar to original paper @open sesame 04/20 15:09

### PR DESCRIPTION
**Pull request by baek2sm**
Yolo v2 model was updated similar to original yolo v2 model.

This model was intended to be implemented in accordance with
the original paper of Yolo v2 as much as possible,
but now average pooling is temporarily used instead of the
re-organization module.

If only the average pooling is replaced with the re-organization
module in the future, the rest is the same as the original paper
in Yolo v2.

Both the PyTorch version and the NNTrainer version updated the model
structure and verified that the same results could be obtained
by loading trained weights from PyTorch.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

File: Applications/YOLO/jni/main.cpp [hunk 1]
```diff
@@ -177,21 +179,50 @@ ModelHandle YOLO() {
   blocks.push_back(yoloBlock("conv7", "conv6", 128, 1, false));
   blocks.push_back(yoloBlock("conv8", "conv7", 256, 3, true));
   blocks.push_back(yoloBlock("conv9", "conv8", 512, 3, false));
-  blocks.push_back(yoloBlock("conv10", "conv9", 256, 1, true));
+  blocks.push_back(yoloBlock("conv10", "conv9", 256, 1, false));
+  blocks.push_back(yoloBlock("conv11", "conv10", 512, 3, false));
+  blocks.push_back(yoloBlock("conv12", "conv11", 256, 1, false));
+  blocks.push_back(yoloBlock("conv13", "conv12", 512, 3, false));
+
+  blocks.push_back({createLayer(
```
Comment (jijoongmoon) [inline Applications/YOLO/jni/main.cpp:187]: plz, leave a comment this will be replaced with residual in a later pr.

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: Applications/YOLO/jni/main.cpp (hunk 1 above)
Comment (baek2sm) [reply Applications/YOLO/jni/main.cpp:187]: thanks, I added a comment at "conv_b_pool(average pooling)", line 200.

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Pull request merged**